sample/golang-echo/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.9-alpine
```

Comment:
Here's a different idea - can we do the sample in a different PR? .. just the custom support in this one and then we can get it merged.